src/main.rs

```diff
@@ -27,8 +31,15 @@ use std::time::{Duration, Instant};
 const FPS_REFRESH_TIMEOUT: u64 = 1;
 const FRAME_RATE_CAP: u32 = 60;
 const PKG_NAME: &'static str = env!("CARGO_PKG_NAME");
+const PKG_VERSION: &'static str = env!("CARGO_PKG_VERSION");
 
 fn main() -> io::Result<()> {
+    logging::init(vec![
+        Box::new(LogFile::new(Level::Debug)),
+        Box::new(LogStdOut::new(Level::Info)),
```

Comment:
what should we do with the log level ? always the same with a verbose flag or should we set it according to debug/release build